### PR TITLE
Enhance display of books loaned by a person

### LINKED ISSUE #149: Create highlighted field around loaned books under User

### PR DESCRIPTION
**Pull request by TYKCodes**
Resolves #149 
The displaying of loaned books for each person is enhanced to match the design of tags, instead of just displaying a long string of the book details.

Issues: 
- Currently trying to find a way to align all the Book labels to the start of the label "Books loaned: "

Any feedback is appreciated! If the design is too glaring or does not fit well, please provide comments below.

### CODE REVIEW

**Review by sprintaway — APPROVED** [review]
LGTM

**Comment by sprintaway** [comment]
Btw what does this issue mean:
Currently trying to find a way to align all the Book labels to the start of the label "Books loaned: "

And also can maybe consider adding "None" after Books loaned, or just hide the Books loaned part if a person has no book on loan to make it look more tidy.

**Comment by TYKCodes** [comment]
Thank you for the approval! I will probably be adding some updates to the PR based on the comments made, so I will not merge the PR yet.

> Btw what does this issue mean: Currently trying to find a way to align all the Book labels to the start of the label "Books loaned: "
>
Currently, the book titles are displayed in a newline (eg. Below the "Books loaned: " label) and not inline after the label. 

> And also can maybe consider adding "None" after Books loaned, or just hide the Books loaned part if a person has no book on loan to make it look more tidy.
>
I will probably remove the label because I have not found a solution to print the Books inline with the label

>Another small issue is that now we are unable to see the return date in the user list which may be a problem since different books may have different return dates.
>
I will look into it and see if I can make use of the method that prints out the Loan details and integrate it into the UI

**Review by sprintaway — COMMENTED** [review]
As stated

**Comment by TYKCodes** [comment]
I see... The resizing issue does make the UI look weird.
Okay, I have decided to remove "Books loaned: " regardless; the colour and highlighting of the Loan should be good enough to indicate that it is a Loan detail. If it is okay with you, you may approve the PR. Thank you!

**Review by sprintaway — APPROVED** [review]
LGTM

File: src/main/java/bookface/model/book/Book.java [hunk 1]
```diff
@@ -128,9 +128,9 @@ public int hashCode() {
     @Override
     public String toString() {
         return getTitle()
-                + "; Author: "
+                + " | Author: "
                 + getAuthor()
-                + "; "
+                + " | "
                 + getReturnDateString();
```
Comment (parth-io) [inline src/main/java/bookface/model/book/Book.java:134]: `getReturnDateString` returns null, we should change it to return "Not on loan"

**Pull request merged**